apps/examples/testcase/le_tc/network/itc_net_connect.c

```diff
@@ -35,6 +36,57 @@
 #define CLIENT_WAIT_TIME 7
 
 static int g_port;
+static sem_t g_sig_clitomain;
+static sem_t g_sig_srvtomain;
+
+static int _recv(int sd, char *buf, int buf_size)
+{
+	int recved = 0;
+	while (recved < buf_size) {
+		int ret = recv(sd, buf + recved, buf_size - recved, MSG_WAITALL);
+		if (ret <= 0) {
+			printf("recv error %d %s %d %s:%d\n",ret,  strerror(errno), __FUNCTION__, __LINE__);
```

Comment:
There are five formats `%d %s %d %s:%d`, but there are only four variables.
Please check them.
And please add space after `,` like below.
```
\n", ret, ...
```

Reply:
Thanks I'll modify it